py4DSTEM/visualize/visualize.py

```diff
@@ -413,5 +413,33 @@ def show_points(ar,x,y,s=1,scale=500,alpha=1,point_color='r',
     else:
         return fig,ax
 
-
+    
+def show_hist(arr, bins = 200, return_hist = False):
+    
+    """
+    Visualization function to show histogram from any ndarray (arr).
+    
+    Accepts:
+        arr                 (ndarray)
+        bins                (int) number of bins that the intensity values will be sorted into for histogram
+        return_fig          (bool) 
+    
+    Returns:
+        If return_hist==False (default), the figure is plotted and nothing is returned.
+        If return_hist==True, the counts and bins are returned for custom visualization
+    """
+    
+    counts, bin_values = np.histogram(arr, bins = bins, range = (np.min(arr), np.max(arr)) )
+    bin_values = bin_values[0:bins]
+    
+    fig, ax = plt.subplots(1,1)
+    ax.plot(bin_values, counts)
```

Comment:
This is actually not quite right!  np.histogram returns the bin *edges*.  So your scatterplot is showing histogram values that are all offset to the left by 1/2 the bin width.  Actually, I'm surprised this isn't breaking, as bin_values and counts should have lengths that differ by one...so perhaps I'm missing something here?  Normally to plot this histogram with the values, I'd suggest doing something like
```
counts,bin_edges = np.histogram(...)
bin_width = bin_edges[1]-bin_edges[0]
bin_centers = bin_edges[:-1]+bin_width/2.

fig,ax = plt.subplots()
ax.bar(bin_centers,counts,width=bin_width,align='center')
...
```
Strictly you don't need to specify the align param, as center is default, but explicitness is nice :3